setup/openchange_neworganization

```diff
@@ -41,8 +41,13 @@ parser.add_option("--firstorg", type="string", metavar="FIRSTORG",
 parser.add_option("--firstou", type="string", metavar="FIRSTOU",
                   default="First Administrative Group",
                   help="set OpenChange Administrative Group (otherwise 'First Administrative Group')")
+parser.add_option("--ignore-already-exists", action="store_true",
+                  help="Ignore errors when installing schemas that already exists")
+parser.add_option("--enable-all-groups", action="store_true",
```

Comment:
I don't understand why put this option in this file instead of `openchange_group`